docs/source-app/workflows/encrypted_secrets/secrets.rst

```diff
@@ -0,0 +1,31 @@
+.. _secrets:
+
+#################
+Encrypted Secrets
+#################
+
+Is your App using data or values (for example: API keys or access credentials) that you don't want to expose in your App code? If the answer is yes, you'll want to use Secrets. Secrets are encrypted values that are stored in the Lightning.ai database and are decrypted at runtime.
+
+.. tip::
+	For non-sensitive configuration values, we recommend using :ref:`plain-text Environment Variables <environment_variables>`.
+
+***************
+What did we do?
+***************
+
+When a Lightning App (App) **runs in the cloud**, a Secret can be exposed to the App using environment variables.
```

Comment:
can you add an example to show how to access them?

Reply:
The example is in the section below.